packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -410,12 +410,6 @@ export class FASTSlider extends FormAssociatedSlider implements SliderConfigurat
      *  If the event handler is null it removes the events
      */
     private handleThumbMouseDown = (event: MouseEvent | null): void => {
-        if (event) {
-            if (this.readOnly || this.disabled || event.defaultPrevented) {
```

Comment:
@eljefe223 One catch - we should probably keep the check here and return if it's readonly/disabled/etc. We should not focus, but I think this guard is good.

Reply:
handleMouseMove and handleMouseDown already catch disabled which are the events this method is adding, I think if we want to be safe we can add the disabled check alongside `readOnly` on line 293 in handleKeyDown. I think we had redundant checking.